src/main/java/org/candlepin/service/CloudRegistrationAdapter.java

```diff
@@ -30,18 +31,41 @@ public interface CloudRegistrationAdapter {
      * Resolves the cloud registration details to a specific organization.
      *
      * @param cloudRegInfo
-     *  A CloudRegistrationInfo instance which contains the cloud provider details to process
+     *     A CloudRegistrationInfo instance which contains the cloud provider details to process
      *
      * @throws MalformedCloudRegistrationException
-     *  if the cloud registration info is null, incomplete, or invalid
+     *     if the cloud registration info is null, incomplete, or invalid
      *
      * @throws CloudRegistrationAuthorizationException
-     *  if cloud registration is not permitted for the provider or account holder
+     *     if cloud registration is not permitted for the provider or account holder
      *
-     * @return
-     *  the owner key of the owner (organization) to which the cloud user will be registered
+     * @return the owner key of the owner (organization) to which the cloud user will be registered
      */
     String resolveCloudRegistrationData(CloudRegistrationInfo cloudRegInfo)
         throws CloudRegistrationAuthorizationException, MalformedCloudRegistrationException;
 
+    /**
+     * Create and entitle isAnonymous organization.
```

Comment:
Something more accurate would be: `Create (if one does not already exist) and entitle an owner upstream of Candlepin, for the given cloud account id and cloud offering id.`

Reply:
done